### PR TITLE
{data}[gompi/2023b] CDO v2.2.2, UDUNITS v2.2.28 , PROJ v9.3.1, ecCodes v2.31.0 , libaec v1.0.6, nlohmann_json v3.11.3, pkgconfig v1.5.5

### PR DESCRIPTION
**Pull request by TopRichard**

### CODE REVIEW

**Comment by TopRichard** [comment]
@smoors Do these easyconfigs require any modifications?

File: easybuild/easyconfigs/c/CDO/CDO-2.2.2-gompi-2023b.eb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    ('util-linux', '2.39'),
+]
+
+# Build libcdi
+configopts = "--enable-cdi-lib "
+
+# Use dependencies from EasyBuild
+configopts += "--with-curl=$EBROOTCURL --with-eccodes=$EBROOTECCODES --with-fftw3 --with-hdf5=$EBROOTHDF5 "
+configopts += "--with-netcdf=$EBROOTNETCDF --with-proj=$EBROOTPROJ --with-szlib=$EBROOTSZIP "
+configopts += "--with-udunits2=$EBROOTUDUNITS --with-util-linux-uuid=$EBROOTUTILMINLINUX "
```
Comment (boegel) [inline easybuild/easyconfigs/c/CDO/CDO-2.2.2-gompi-2023b.eb:44]: small extra bit needed here to make sure that build on CDO works fine on non-x86_64 systems, see #19735

```suggestion
configopts += "--with-udunits2=$EBROOTUDUNITS --with-util-linux-uuid=$EBROOTUTILMINLINUX "

# Make sure that right Fortran compiler is used, also on non-x86_64 architectures
configopts += 'CPPFLAGS="$CPPFLAGS -DgFortran" '

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso
CORE_CNT=16

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3
CORE_CNT=16

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in total)
node3110.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/f03e6791e0e3125585b2bae9dd3b01f1 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @TopRichard!

**Pull request merged**